.github/workflows/verify.yaml

```diff
@@ -7,11 +7,6 @@ on:
     paths-ignore:
       - '**.md'
       - 'docs/**'
-# Grant permissions to obtain federated identity credentials
```

Comment:
I know very little about these permissions, but would removing them from the workflow level not mean that _no_ jobs have the permissions, except `Check-Cloud-Environment` and `Upload-Test-Report`?

Reply:
@paullatzelsperger by default the permissions provided by `GITHUB-TOKEN` are used, `permissions` permit to add or remove access.
In my opinion setting them at the global level was not the best choice, since every job should have their custom permissions set.
https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs